llama_cpp/llama.py

```diff
@@ -1363,6 +1414,11 @@ def logit_bias_processor(
                 break
 
             if stream:
+                if stream_options is not None and "include_usage" in stream_options:
+                    include_usage = True if stream_options["include_usage"] else False
+                else:
+                    include_usage = False    
```

Comment:
Instead of the double nested if block, presumably you could do: `if stream and stream_options and "include_usage" in stream_options:`? @tpfau

Reply:
No you can't since all the following code has to be run in both instances, and only the include stream usage needs to be adapted based on the props.

Reply:
Ahh, of course. I guess I was meant something like this:
```python
if stream_options and "include_usage" in stream_options:
    include_usage = stream_options["include_usage"]
else:
    include_usage = False
```
It relies on a None `stream_options` being _falsey_, and `stream_options.include_usage` being a boolean.
Just a little more readable.